api/buckets/pb/javascript/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@textile/buckets",
+  "name": "@textile/buckets-grpc",
   "version": "0.0.0",
-  "description": "A client for interacting with the Textile Buckets gRPC API.",
+  "description": "Interfaces/classes for interacting with the Textile Buckets gRPC API.",
```

Comment:
It's a client for interacting with the gRPC API, no?

Reply:
Is it? These don't include the client libs, just the proto classes etc right? Or maybe I have an overly strict definition of client?

Reply:
I would consider the thing in @textile/textile the _client_.